packages/core/tests/integration/components/navi-visualizations/goal-gauge-test.ts

```diff
@@ -0,0 +1,364 @@
+import { A as arr } from '@ember/array';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { findAll, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { TestContext as Context } from 'ember-test-helpers';
+import StoreService from '@ember-data/store';
+import GoalGaugeVisualization from 'navi-core/components/navi-visualizations/goal-gauge';
+import NaviFactResponse from 'navi-data/models/navi-fact-response';
+import RequestFragment from 'navi-core/models/bard-request-v2/request';
+
+const TEMPLATE = hbs`
+<NaviVisualizations::GoalGauge
+  @model={{this.model}}
+  @options={{this.options}}
+/>`;
+
+type ComponentArgs = GoalGaugeVisualization['args'];
+
+interface TestContext extends Context, ComponentArgs {
+  request: RequestFragment;
+}
+
+module('Integration | Component | navi-visualization/goal gauge ', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function(this: TestContext) {
+    const store = this.owner.lookup('service:store') as StoreService;
```

Comment:
do we need to load the metadata?

Reply:
Currently I do not think we use it, but I could add it in if you like?